src/replicache.ts

```diff
@@ -78,6 +78,12 @@ export interface ReplicacheOptions {
    */
   name?: string;
 
+  /**
+   * The schema version of the data understood by this application. This allows
+   * your backend to know what format to return in the pull requests.
```

Comment:
Maybe be a little bit more specific re how it is used? .... This enables versioning of mutators (in the push direction) and the client view (in the pull direction).